metricflow/model/data_warehouse_model_validator.py

```diff
@@ -341,25 +359,6 @@ def gen_measure_tasks(
                         error_message=f"Unable to query measure `{spec.element_name}` on data source `{data_source.name}` in data warehouse",
                     )
                 )
-
-            filter_elements_node = FilterElementsNode(parent_node=source_node, include_specs=data_source_specs)
```

Comment:
I think we need to make the filter specs be the union of all measure specs here and keep the other task in but I admit I'm guessing a bit.